sso/sso72-x509-https.json

```diff
@@ -183,25 +159,6 @@
                 }
             }
         },
-        {
-            "kind": "Route",
-            "apiVersion": "v1",
-            "id": "${APPLICATION_NAME}-http",
-            "metadata": {
-                "name": "${APPLICATION_NAME}",
```

Comment:
As you're removing this route, would you want to update the name of the https rout to just `${APPLICATION_NAME}` instead of `secure-${APPLICATION_NAME}`?

Reply:
I can. But wouldn't this cause confusion in ```passthrough``` templates the HTTPs route is named  ```secure-${APPLICATION_NAME}```, and in x509 ones it would be ```${APPLICATION_NAME}```?

(IOW the same ```${APPLICATION_NAME}``` would be use to denote HTTP route in ```passthrough``` templates, but for HTTPS route in ```x509``` templates)

Reply:
Isn't the goal to simplify things?  The only reason we have different names is because you can't use the same name for both http and https. :/  If we only have one route, we should use the most intuitive route name.

Reply:
Ok, fair enough. Will update the names.